composer.json

```diff
@@ -42,7 +42,8 @@
             "src/Core/functions.php",
             "src/Collection/functions.php",
             "src/I18n/functions.php",
-            "src/Utility/bootstrap.php"
+            "src/Utility/bootstrap.php",
+            "src/TestSuite/phpunit-forward-compat.php"
```

Comment:
Why was this PR merged though @antograssiot agreed to pepare something else? 😕 

Also couldn't ``src/TestSuite/phpunit-forward-compat.php`` be loaded through ``autoload-dev``?
https://getcomposer.org/doc/04-schema.md#autoload-dev